src/main/java/com/example/java/test/junior/developer/security/KeycloakAdminClient.java

```diff
@@ -0,0 +1,17 @@
+package com.example.java.test.junior.developer.security;
+
+import lombok.Data;
+import org.keycloak.admin.client.Keycloak;
+import org.keycloak.representations.idm.UserRepresentation;
+import org.springframework.context.annotation.Bean;
+
+@Data
```

Comment:
What is this annotation doing here?

Reply:
It should be annotated with `@Component` right? This is part of our spring context.